### ISSUE TITLE
Outlook add-in with SupportsMultiSelect feature is not working for personal MS accounts

### ISSUE THREAD

**Issue by rohanbharadwaj1994** [OfficeDev/office-js#5046]
# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the Web
* Host [Excel, Word, PowerPoint, etc.]: Outlook web
* Office version number: 
* Operating System: Mac OS
* Browser (if using Office on the web): Chrome
 
## Expected behavior

Add-in should work for personal MS accounts as well after adding SupportsMultiSelect element to true in the manifest

## Current behavior

When SupportsMultiSelect added in the manifest and deployed, it should support multi selection feature and add-in should also work fine. it is working fine on all the platforms. But for Personal MS account (created from gmail) on https://outlook.live.com/. The add-in is not getting opened up when triggered from the touch point.

## Steps to reproduce

1. Add support for Multiselect in outlook add-in XML manifest: https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/item-multi-select?tabs=jsonmanifest
2. Deploy the manifest on any microsoft personal account (outlook.live.com) (Not the enterprise one)
3. Trigger the add-in from the touch point Read view and see that the add-in is not opening up.

## Context

This issue is reproducible for personal accounts

## Useful logs
[screen-capture (41).webm](https://github.com/user-attachments/assets/5f307d60-3c02-4171-a16e-3b5736199b68)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by rohanbharadwaj1994** [comment]
Hi,
Any update regarding this issue ??

**Comment by mmanjaree-msft** [comment]
Can you share logs in a private repo- [Collecting log files for Outlook Add‐ins issues](https://github.com/OfficeDev/office-js/wiki/Collecting-log-files-for-Outlook-Add%E2%80%90ins-issues).
Also is the issue there with a personal account created originally in outlook?

**Comment by rohanbharadwaj1994** [comment]
Yes @mmanjaree-msft The personal account has been created originally in outlook

**Comment by mobisw-msft** [comment]
Thank you for reporting this issue regarding MultiSelect support for Live accounts. It has been put in our backlog. Unfortunately we have no timelines to share at this point.

Internal Tracking ID: 5060374

**Comment by mobisw-msft** [comment]
Hey @rohanbharadwaj1994 ,

This issue has now been fixed. Please do confirm this behaviour, and we will reopen the issue if this behaviour is not fixed.

**Comment by mobisw-msft** [comment]
Hey @rohanbharadwaj1994 ,

This issue has now been fixed and you should be able to see the fixed behavior. Please reopen this issue if this still persists.
